cli/status/lua/instance_state.lua

```diff
@@ -0,0 +1,57 @@
+local ok, config = pcall(require, 'config')
+if ok and config ~= nil and type(config) == "table" and type(config.info) == "function" then
+    config = config:info()
+else
+    config = {
+        status = "uninitialized",
+        alerts = {},
+    }
+end
+
+local config_info = {
+    status = config.status,
+    alerts = {}
+}
+for _, alert in pairs(config.alerts) do
+    table.insert(config_info.alerts, {
+        type = alert.type,
+        message = alert.message
+    })
+end
+
+local ok, box_replication = pcall(function() return box.info.replication end)
+if not ok or box_replication == nil then
+    box_replication = {}
+end
+
+local replication_info = {}
+for _, instance in ipairs(box_replication) do
```

Comment:
The keys in this table are instance IDs ([code](https://github.com/tarantool/tarantool/blob/8fae8004f79ecd555537960c60c6e646b037c4cc/src/box/lua/info.c#L228-L236) and I guess there is no guarantee that there are no holes in this table if we consder it as an array. It is better to use `pairs` here.